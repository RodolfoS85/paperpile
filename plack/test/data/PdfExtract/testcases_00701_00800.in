---
file: ../../../PDFREPOSITORY/000701.pdf
---
file: ../../../PDFREPOSITORY/000702.pdf
---
file: ../../../PDFREPOSITORY/000703.pdf
---
file: ../../../PDFREPOSITORY/000704.pdf
---
file: ../../../PDFREPOSITORY/000705.pdf
---
file: ../../../PDFREPOSITORY/000706.pdf
---
file: ../../../PDFREPOSITORY/000707.pdf
---
file: ../../../PDFREPOSITORY/000708.pdf
---
file: ../../../PDFREPOSITORY/000709.pdf
---
file: ../../../PDFREPOSITORY/000710.pdf
---
file: ../../../PDFREPOSITORY/000711.pdf
---
file: ../../../PDFREPOSITORY/000712.pdf
---
file: ../../../PDFREPOSITORY/000713.pdf
---
file: ../../../PDFREPOSITORY/000714.pdf
---
file: ../../../PDFREPOSITORY/000715.pdf
---
file: ../../../PDFREPOSITORY/000716.pdf
---
file: ../../../PDFREPOSITORY/000717.pdf
---
file: ../../../PDFREPOSITORY/000718.pdf
---
file: ../../../PDFREPOSITORY/000719.pdf
---
file: ../../../PDFREPOSITORY/000720.pdf
---
file: ../../../PDFREPOSITORY/000721.pdf
---
file: ../../../PDFREPOSITORY/000722.pdf
---
file: ../../../PDFREPOSITORY/000723.pdf
---
file: ../../../PDFREPOSITORY/000724.pdf
---
file: ../../../PDFREPOSITORY/000725.pdf
---
file: ../../../PDFREPOSITORY/000726.pdf
---
file: ../../../PDFREPOSITORY/000727.pdf
---
file: ../../../PDFREPOSITORY/000728.pdf
---
file: ../../../PDFREPOSITORY/000729.pdf
---
file: ../../../PDFREPOSITORY/000730.pdf
---
file: ../../../PDFREPOSITORY/000731.pdf
---
file: ../../../PDFREPOSITORY/000732.pdf
---
file: ../../../PDFREPOSITORY/000733.pdf
---
file: ../../../PDFREPOSITORY/000734.pdf
---
file: ../../../PDFREPOSITORY/000735.pdf
---
file: ../../../PDFREPOSITORY/000736.pdf
---
file: ../../../PDFREPOSITORY/000737.pdf
---
file: ../../../PDFREPOSITORY/000738.pdf
---
file: ../../../PDFREPOSITORY/000739.pdf
---
file: ../../../PDFREPOSITORY/000740.pdf
---
file: ../../../PDFREPOSITORY/000741.pdf
---
file: ../../../PDFREPOSITORY/000742.pdf
---
file: ../../../PDFREPOSITORY/000743.pdf
---
file: ../../../PDFREPOSITORY/000744.pdf
---
file: ../../../PDFREPOSITORY/000745.pdf
---
file: ../../../PDFREPOSITORY/000746.pdf
---
file: ../../../PDFREPOSITORY/000747.pdf
---
file: ../../../PDFREPOSITORY/000748.pdf
---
file: ../../../PDFREPOSITORY/000749.pdf
---
file: ../../../PDFREPOSITORY/000750.pdf
---
file: ../../../PDFREPOSITORY/000751.pdf
---
file: ../../../PDFREPOSITORY/000752.pdf
---
file: ../../../PDFREPOSITORY/000753.pdf
---
file: ../../../PDFREPOSITORY/000754.pdf
---
file: ../../../PDFREPOSITORY/000755.pdf
---
file: ../../../PDFREPOSITORY/000756.pdf
---
file: ../../../PDFREPOSITORY/000757.pdf
---
file: ../../../PDFREPOSITORY/000758.pdf
---
file: ../../../PDFREPOSITORY/000759.pdf
---
file: ../../../PDFREPOSITORY/000760.pdf
---
file: ../../../PDFREPOSITORY/000761.pdf
---
file: ../../../PDFREPOSITORY/000762.pdf
---
file: ../../../PDFREPOSITORY/000763.pdf
---
file: ../../../PDFREPOSITORY/000764.pdf
---
file: ../../../PDFREPOSITORY/000765.pdf
---
file: ../../../PDFREPOSITORY/000766.pdf
---
file: ../../../PDFREPOSITORY/000767.pdf
---
file: ../../../PDFREPOSITORY/000768.pdf
---
file: ../../../PDFREPOSITORY/000769.pdf
---
file: ../../../PDFREPOSITORY/000770.pdf
---
file: ../../../PDFREPOSITORY/000771.pdf
---
file: ../../../PDFREPOSITORY/000772.pdf
---
file: ../../../PDFREPOSITORY/000773.pdf
---
file: ../../../PDFREPOSITORY/000774.pdf
---
file: ../../../PDFREPOSITORY/000775.pdf
---
file: ../../../PDFREPOSITORY/000776.pdf
---
file: ../../../PDFREPOSITORY/000777.pdf
---
file: ../../../PDFREPOSITORY/000778.pdf
---
file: ../../../PDFREPOSITORY/000779.pdf
---
file: ../../../PDFREPOSITORY/000780.pdf
---
file: ../../../PDFREPOSITORY/000781.pdf
---
file: ../../../PDFREPOSITORY/000782.pdf
---
file: ../../../PDFREPOSITORY/000783.pdf
---
file: ../../../PDFREPOSITORY/000784.pdf
---
file: ../../../PDFREPOSITORY/000785.pdf
---
file: ../../../PDFREPOSITORY/000786.pdf
---
file: ../../../PDFREPOSITORY/000787.pdf
---
file: ../../../PDFREPOSITORY/000788.pdf
---
file: ../../../PDFREPOSITORY/000789.pdf
---
file: ../../../PDFREPOSITORY/000790.pdf
---
file: ../../../PDFREPOSITORY/000791.pdf
---
file: ../../../PDFREPOSITORY/000792.pdf
---
file: ../../../PDFREPOSITORY/000793.pdf
---
file: ../../../PDFREPOSITORY/000794.pdf
---
file: ../../../PDFREPOSITORY/000795.pdf
---
file: ../../../PDFREPOSITORY/000796.pdf
---
file: ../../../PDFREPOSITORY/000797.pdf
---
file: ../../../PDFREPOSITORY/000798.pdf
---
file: ../../../PDFREPOSITORY/000799.pdf
---
file: ../../../PDFREPOSITORY/000800.pdf
