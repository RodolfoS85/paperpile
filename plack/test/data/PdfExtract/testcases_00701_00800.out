---
authors: 'Konermann, Lars and Simmons, Douglas A.'
title: PROTEIN-FOLDING KINETICS AND MECHANISMS STUDIED BY PULSE-LABELING AND MASS SPECTROMETRY
---
authors: 'Koonin, Eugene V. and Wolf, Yuri I. and Karev, Georgy P.'
title: The structure of the protein universe and genome evolution
---
authors: 'Kornberg, Thomas B.'
title: Understanding the Homeodomain
---
authors: 'Kornblihtt, Alberto R'
title: 'Chromatin, transcript elongation and alternative splicing'
---
authors: 'Kornblihtt, Alberto R. and DE LA MATA, Manuel and Fededa, Juan Pablo and MUÑOZ, Manuel J. and NOGUÉS, Guadalupe'
doi: 10.1261/rna.7100104
title: Multiple links between transcription and splicing
---
authors: 'Korotkov, Konstantin V. and Novoselov, Sergey V. and Hatfield, Dolph L. and Gladyshev, Vadim N.'
doi: 10.1128/MCB.22.5.1402-1411.2002
title: Mammalian Selenoprotein in Which Selenocysteine (Sec) Incorporation Is Supported by a New Form of Sec Insertion Sequence Element
---
authors: 'Kostrzewa, Markus and Muller, Ulrich'
title: Genomic structure and complete sequence of the human FGFR4 gene
---
authors: 'Kozak, M.'
title: 'Interpreting cDNA sequences: some insights from studies on translation'
---
authors: 'Kozak, Marilyn'
doi: 10.1128/MCB.21.6.1899-1907.2001
title: New Ways of Initiating Translation in Eukaryotes?
---
authors: 'Kozak, Marilyn'
title: Pushing the limits of the scanning mechanism for initiation of translation
---
authors: 'Kozak, Marilyn'
doi: 10.1016/j.gene.2005.06.037
title: Regulation of translation via mRNA structure in prokaryotes and eukaryotes
---
authors: 'Kraus, Richard J. and Murray, Elizabeth E. and Wiley, Steven R. and Zink, Nancy M. and Loritz, Karla and Gelembiuk, Gregory W. and Mertz, Janet E.'
title: Experimentally determined weight matrix definitions of the initiator and TBP binding site elements of promoters
---
authors: 'Kretschmann, Ernst and Fleischmann, Wolfgang and Apweiler, Rolf'
title: Automatic rule generation for protein annotation with the C4.5 data mining algorithm applied on SWISS-PROT
---
authors: 'Krippahl, Ludwig and Barahona, Pedro'
title: Applying Constraint Programming to Protein Structure Determination
---
authors: 'Krippahl, Ludwig and Moura, Jose J. and Nuno Palma, P.'
title: Modeling Protein Complexes with BiGGER
---
authors: 'Krogh, Anders and Larsson, Bjorn and von Heijne, Gunnar and Sonnhammer, Erik L. L.'
doi: 10.1006/jmbi.2000.4315
title: 'Predicting Transmembrane Protein Topology with a Hidden Markov Model: Application to Complete Genomes'
---
authors: 'Kruspe, Matthias and Stadler, Peter F'
doi: 10.1186/1471-2105-8-254
title: Progressive multiple sequence alignments from triplets
---
authors: 'Kryukov, Gregory V. and Kryukov, Valentin M. and Gladyshev, Vadim N.'
title: New Mammalian Selenocysteine-containing Proteins Identified with an Algorithm That Searches for Selenocysteine Insertion Sequence Elements
---
authors: 'Kschischo, Maik and Lassig, Michael'
title: Finite-temperature Sequence Alignment
---
authors: 'Kschischo, Maik and Lässig, Michael and Yu, Yi-Kuo'
doi: 10.1016/j.bulm.2004.07.001
title: Toward an accurate statistics of gapped alignments
---
authors: 'Kubarenko, Andriy V. and Sergiev, Petr V. and Bogdanov, Alexey A. and Brimacombe, Richard and Dontsova, Olga A.'
title: A protonated base pair participating in rRNA tertiary structural interactions
---
authors: 'Kuhlmann, Marco and Möhl, Mathias and Lab, Programming Systems'
title: Extended cross-serial dependencies in Tree Adjoining Grammars
---
authors: 'Kuhlmann, Marco and Lab, Programming Systems'
title: Mildly Context-Sensitive Dependency Languages
---
authors: 'Kuhner, Mary K. and Felsenstein, Joseph'
title: A Simulation Comparison of Phylogeny Algorithms under Equal and Unequal Evolutionary Rates
---
authors: 'Kulinski, Tadeusz and Olejniczak, Mikolaj and Huthoff, Hendrik and Bielecki, Lukasz and Pachulska-Wieczorek, Katarzyna and Das, Atze T. and Berkhout, Ben and Adamiak, Ryszard W.'
doi: 10.1074/jbc.M301939200
title: The Apical Loop of the HIV-1 TAR RNA Hairpin Is Stabilized by a Cross-loop Base Pair
---
authors: 'Kumar, Sandeep and Ma, Buyong and Tsai, CHUNG-JUNG and Sinha, Neeti and Nussinov, Ruth'
title: 'Folding and binding cascades: Dynamic landscapes and population shifts'
---
authors: 'Kumar, Sudhir and Gadagkar, Sudhindra R.'
doi: 10.1007/s002390010118
title: Efficiency of the Neighbor-Joining Method in Reconstructing Deep and Shallow Evolutionary Relationships in Large Phylogenies
---
authors: 'Kundaje, Anshul and Middendorf, Manuel and Shah, Mihir and Wiggins, Chris H and Freund, Yoav and Leslie, Christina'
doi: 10.1186/1471-2105-7-S1-S5
title: A classification-based framework for predicting and analyzing gene regulatory response
---
authors: 'Kuo, Byron Yu-Lin and Chen, Ying and Bohacec, Slavita and Johansson, Ojvind and Wasserman, Wyeth W. and Simpson, Elizabeth M.'
doi: 10.1371/journal.pcbi.0020034
title: 'SAGE2Splice: Unmapped SAGE Tags Reveal Novel Splice Junctions'
---
authors: 'Kuroda, Takao and Tada, Masako and Kubota, Hiroshi and Kimura, Hironobu and Hatano, Shin-ya and Suemori, Hirofumi and Nakatsuji, Norio and Tada, Takashi'
doi: 10.1128/MCB.25.6.2475-2485.2005
title: Octamer and Sox Elements Are Required for Transcriptional cis Regulation of Nanog Gene Expression
---
authors: 'Kwon, Chulan and Han, Zhe and Olson, Eric N. and Srivastava, Deepak'
doi: 10.1073/pnas.0509535102
title: MicroRNA1 influences cardiac differentiation in Drosophila and regulates Notch signaling
---
authors: 'Ladurner, Andreas G. and Fersht, Alan R.'
title: 'Glutamine, Alanine or Glycine Repeats Inserted into the Loop of a Protein Have Minimal Effects on Stability and Folding Rates'
---
authors: 'Lafontaine, Daniel A. and Norman, David G. and Lilley, David M. J.'
title: 'Structure, folding and activity of the VS ribozyme: importance of the 2-3-6 helical junction'
---
authors: 'Lai, Eric C and Tomancak, Pavel and Williams, Robert W and Rubin, Gerald M'
title: Computational identification of Drosophila microRNA genes
---
authors: 'Lai, Eric C. and Tam, Bergin and Rubin, Gerald M.'
doi: 10.1101/gad.1291905
title: 'Pervasive regulation of Drosophila Notch target genes by GY-box-, Brd-box-, and K-box-class microRNAs'
---
authors: 'Lam, Bianca J. and Hertel, Klemens J.'
title: A general role for splicing enhancers in exon definition
---
authors: 'Lambert, André and Lescure, Alain and Gautheret, Daniel'
title: A survey of metazoan selenocysteine insertion sequences
---
authors: 'Landau, Gad M.'
title: On the Common Substring Alignment Problem
---
authors: 'Lander, Eric S.'
title: Array of hope
---
authors: 'Langmead, Christopher James and Donald, Bruce Randall'
title: Extracting Structural Information Using Time-Frequency Analysis of Protein NMR Data
---
authors: 'Langmead, Ben and Trapnell, Cole and Pop, Mihai and Salzberg, Steven L'
doi: 10.1186/gb-2009-10-3-r25
title: Ultrafast and memory-efficient alignment of short DNA sequences to the human genome
---
authors: 'Lankas, Filip and Sponer, Ïõ and Hobza, Pavel and Langowski, Jorg'
doi: 10.1006/jmbi.2000.3781
title: Sequence-dependent Elastic Properties of DNA
---
authors: 'Lao, Demelo Madrazo and Shimizu, Toshio'
title: Methods for Detecting the Signal Peptide in Transmembrane and Globular Proteins
---
authors: 'Latchman, David S.'
title: Eukaryotic transcription factors
---
authors: 'Lathrop, Richard H. and Smith, Temple F.'
title: Global Optimum Protein Threading with Gapped Alignment and Empirical Pair Score Functions
---
authors: 'Laughton, Charles and Luisi, Ben'
title: 'The Mechanics of Minor Groove Width Variation in DNA, and its Implications for the Accommodation of Ligands'
---
authors: 'Laursen, Brian Søgaard and Sørensen, Hans Peter and Mortensen, Kim Kusk and Sperling-Petersen, Hans Uffe'
doi: 10.1128/MMBR.69.1.101-123.2005
title: Initiation of Protein Synthesis in Bacteria
---
authors: 'Lavorgna, Giovanni and Guffanti, Alessandro and Borsani, Giuseppe and Ballabio, Andrea and Boncinelli, Edoardo'
title: 'TargetFinder: searching annotated sequence databases for target genes of transcription factors'
---
authors: 'Lazaridis, Themis'
title: Effective Energy Function for Proteins in Lipid Membranes
---
authors: 'Lazaridis, Themis and Karplus, Martin'
title: Effective Energy Function for Proteins in Solution
---
authors: 'Lazaridis, Themis and Karplus, Martin'
title: Effective energy functions for protein structure prediction
---
authors: 'Le, Shu-Yun'
title: An algorithm for detecting homologues of known structured RNAs in genomes
---
authors: 'Le, Shu-Yun and Maizel, Jr., Jacob V.'
title: A common RNA structural motif involved in the internal initiation of translation of cellular mRNAs
---
authors: 'Le, SHU-YUN and Maizel, Jr., Jacob V.'
title: Evolution of a Common Structural Core in the Internal Ribosome Entry Sites of Picornavirus
---
authors: 'Le Guiner, Caroline and Plet, Ariane and Galiana, Delphine and Gesnel, Marie-Claude and Del Gatto-Konczak, Fabienne and Breathnach, Richard'
doi: 10.1074/jbc.M107381200
title: Polypyrimidine Tract-binding Protein Represses Splicing of a Fibroblast Growth Factor Receptor-2 Gene Alternative Exon through Exon Sequences
---
authors: 'Le Quesne, John P. C. and Stoneley, Mark and Fraser, Graham A. and Willis, Anne E.'
doi: 10.1006/jmbi.2001.4745
title: Derivation of a Structural Model for the c-myc IRES
---
authors: 'Leaman, Dan and Chen, Po Yu and Fak, John and Yalcin, Abdullah and Pearce, Michael and Unnerstall, Ulrich and Marks, Debora S. and Sander, Chris and Tuschl, Thomas and Gaul, Ulrike'
doi: 10.1016/j.cell.2005.04.016
title: Antisense-Mediated Depletion Reveals Essential and Specific Functions of MicroRNAs in Drosophila Development
---
authors: 'Lease, Richard A. and Belfort, Marlene'
doi: 10.1073/pnas.170281497
title: 'A trans-acting RNA as a control switch in Escherichia coli: DsrA modulates function by forming alternative structures'
---
authors: 'Lease, Richard A. and Cusick, Michael E. and Belfort, Marlene'
title: 'Riboregulation in Escherichia coli: DsrA RNA acts by RNA:RNA interactions at multiple loci'
---
authors: 'Lecuyer, Eric and Yoshida, Hideki and Parthasarathy, Neela and Alm, Christina and Babak, Tomas and Cerovina, Tanja and Hughes, Timothy R. and Tomancak, Pavel and Krause, Henry M.'
doi: 10.1016/j.cell.2007.08.003
title: Global Analysis of mRNA Localization Reveals a Prominent Role in Organizing Cellular Architecture and Function
---
authors: 'Lee, Christopher and Grasso, Catherine and Sharlow, Mark F.'
title: Multiple sequence alignment using partial order graphs
---
authors: 'Lee, June Hyung and Kim, Hana and Ko, Jaehyeong and Lee, Younghoon'
title: Interaction of C5 protein with RNA aptamers selected by SELEX
---
authors: 'Lee, Christopher and Roy, Meenakshi'
title: 'Analysis of alternative splicing with microarrays: successes and challenges'
---
authors: 'Legendre, Matthieu and Gautheret, Daniel'
title: Sequence determinants in human polyadenylation site selection
---
authors: 'Legrain, Pierre and Jestin, Jean-Luc and Schächter, Vincent'
title: From the analysis of protein complexes to proteome-wide linkage maps
---
authors: 'Lenhard, Boris and Sandelin, Albin and Mendoza, Luis and Engström, Pär and Jareborg, Niclas and Wasserman, Wyeth W'
title: Identification of conserved regulatory elements by comparative genome analysis
---
authors: 'Lenhard, Boris and Wasserman, Wyeth W.'
title: 'TFBS: Computational framework for transcription factor binding site analysis'
---
authors: 'Lenski, Richard E. and Ofria, Charles and Pennock, Robert T. and Adami, Christoph'
title: The evolutionary origin of complex features
---
authors: 'Leontis, N. B. and Stombaugh, J. and Westhof, E.'
title: Motif prediction in ribosomal RNAs Lessons and prospects for automated motif prediction in homologous RNA molecules
---
authors: 'Leontis, Neocles B. and Westhof, Eric'
title: Geometric nomenclature and classification of RNA base pairs
---
authors: 'Leopold, Peter E. and Montal, Mauricio and Onuchic, Jost Nelson'
title: 'Protein folding funnels: A kinetic approach to the sequence-structure relationship'
---
authors: 'Lescure, Alain and Fagegaltier, Delphine and Carbon, Philippe and Krol, Alain'
title: Protein Factors Mediating Selenoprotein Synthesis
---
authors: 'Lescure, Alain and Gautheret, Daniel and Carbon, Philippe and Krol, Alain'
title: Novel Selenoproteins Identified in Silico and in Vivo by Using a Conserved RNA Structural Motif
---
authors: 'Leung, Garmay and Eisen, Michael B.'
doi: 10.1371/journal.pone.0006901
title: Identifying Cis-Regulatory Sequences by Word Profile Similarity
---
authors: 'Levanon, Erez Y and Eisenberg, Eli and Yelin, Rodrigo and Nemzer, Sergey and Hallegger, Martina and Shemesh, Ronen and Fligelman, Zipora Y and Shoshan, Avi and Pollock, Sarah R and Sztybel, Dan and Olshansky, Moshe and Rechavi, Gideon and Jantsch, Michael F'
doi: 10.1038/nbt996
title: Systematic identification of abundant A-to-I editing sites in the human transcriptome
---
authors: 'Levine, Aaron and Durbin, Richard'
title: A computational scan for U12-dependent introns in the human genome sequence
---
authors: 'Levine, Michael and Tjian, Robert'
title: Transcription regulation and animal diversity
---
authors: 'Levy, S. and Compagnoni, L. and Myers, E. W. and Stormo, G. D.'
title: 'Xlandscape: the graphical display of word frequencies in sequences'
---
authors: 'Levy, Samuel and Hannenhalli, Sridhar and Workman, Christopher'
title: Enrichment of regulatory signals in conserved non-coding genomic sequence
---
authors: 'Lewis, Benjamin P. and Green, Richard E. and Brenner, Steven E.'
doi: 10.1073/pnas.0136770100
title: Evidence for the widespread coupling of alternative splicing and nonsense-mediated mRNA decay in humans
---
authors: 'Lewis, Benjamin P. and Shih, I-hung and Jones-Rhoades, Matthew W. and Bartel, David P. and Burge, Christopher B.'
title: Prediction of Mammalian MicroRNA Targets
---
authors: 'Li, Ming and Badger, Jonathan H. and Chen, Xin and Kwong, Sam and Kearney, Paul and Zhang, Haoyong'
title: An information-based sequence distance and its application to whole mitochondrial genome phylogeny
---
authors: 'Li, Wei and van Beek, Peter'
title: Guiding Real-World SAT Solving with Dynamic Hypergraph Separator Decomposition
---
authors: 'Li, Yi and Campbell, Colin and Tipping, Michael'
title: Bayesian automatic relevance determination algorithms for classifying gene expression data
---
authors: 'Li, Ming and Chen, Xin and Li, Xin and Ma, Bin and Vitnyi, Paul M. B.'
title: The Similarity Metric
---
authors: 'Li, Jun and Hawkins, Ian C. and Harvey, Christopher D. and Jennings, Jennifer L. and Link, Andrew J. and Patton, James G.'
doi: 10.1128/MCB.23.21.7437-7447.2003
title: Regulation of Alternative Splicing by SRrp86 and Its Interacting Proteins
---
authors: 'Li, Lei and Howe, Gregg A.'
title: Alternative splicing of prosystemin pre-mRNA produces two isoforms that are active as signals in the wound response pathway
---
authors: 'Li, Weizhong and Jaroszewski, Lukasz and Godzik, Adam'
title: Clustering of highly homologous sequences to reduce the size of large protein databases
---
authors: 'Li, Hairi and Lovci, Michael T. and Kwon, Young-Soo and Rosenfeld, Michael G. and Fu, Xiang-Dong and Yeo, Gene W.'
doi: 10.1073/pnas.0807121105
title: 'Determination of tag density required for digital transcriptome analysis: Application to an androgen-sensitive prostate cancer model'
---
authors: 'Li, Renhao and Woodward, Clare'
doi: 10.1110/ps.8.8.1571
title: The hydrogen exchange core and protein folding
---
authors: 'Li, Jinong and Zhang, Zhen and Rosenzweig, Jason and Wang, Young Y. and Chan, Daniel W.'
title: Proteomics and Bioinformatics Approaches for Identification of Serum Biomarkers to Detect Breast Cancer
---
authors: 'Liang, Jie and Dill, Ken A.'
title: Are Proteins Well-Packed?
---
authors: 'Liang, Faming'
doi: 10.1063/1.1387478
title: Evolutionary Monte Carlo for protein folding simulations
---
authors: 'Liao, Guo-chun and Rehm, E. Jay and Rubin, Gerald M.'
doi: 10.1073/pnas.050017397
title: Insertion site preferences of the P transposable element in Drosophila melanogaster
---
authors: 'Li, Chuang and Reches, Myriam and ENGELBERG-KULKA, Hanna'
title: 'The Bulged Nucleotide in the Escherichia coli Minimal Selenocysteine Insertion Sequence Participates in Interaction with SelB: a Genetic Approach'
---
authors: 'Lim, Lee P. and Burge, Christopher B.'
doi: 10.1073/pnas.201407298
title: A computational analysis of sequence features involved in recognition of short introns
---
authors: 'Lin, Jessica and Keogh, Eamonn and Lonardi, Stefano and Patel, Pranav'
title: Finding Motifs in Time Series
---
authors: 'Lin, Meng-Jau and Lee, Tai-Lin and Hsu, Duen-Wei and -K. James Shen, C.'
title: One-codon alternative splicing of the CpG MTase Dnmt1 transcript in mouse somatic cells
---
authors: 'Lin, Li-Hsieh and Lee, Hsiao-Ching and Li, Wen-Hsiung and Chen, Bor-Sen'
doi: 10.1186/1471-2105-6-258
title: Dynamic modeling of cis-regulatory circuits and gene expression prediction via cross-gene identification
---
authors: 'Lincoln, Patrick and Tiwari, Ashish'
title: 'Symbolic Systems Biology: Hybrid Modeling and Analysis of Biological Networks'
